### PR TITLE
wpsoffice: 11.1.0.9080 -> 11.1.0.9505

### LINKED ISSUE #63736: wpsoffice needs update
Current version of wpsoffice in nixpkgs is 10.1.0.5672 while the latest release is 11.1.0.8392 , and the old version package isn't available from official website.

The latest resource could be found [here](http://wps-community.org/downloads). Does anybody update this package?

### PR DESCRIPTION
**Pull request by veprbl**
###### Motivation for this change

Updates wpsoffice to the latest version. Switch to `autoPatchelfHook` should make package more standard. This is based on the hard work done by @Ninlives in #82408.

Closes: #82408
Fixes: #63736

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

cc @Th0rgal

### CODE REVIEW

File: pkgs/applications/office/wpsoffice/default.nix [hunk 1]
```diff
@@ -1,67 +1,112 @@
...
-    libXext
-    fontconfig
-    zlib
-    freetype
-    libICE
-    cups
-    libXrender
-    libxcb
-
+  buildInputs = with xorg; [
```
Comment (worldofpeace) [inline pkgs/applications/office/wpsoffice/default.nix:67]: putting which output here shouldn't be needed, I believe

**Review by Ninlives — APPROVED** [review]
LGTM, but problems listed here https://github.com/NixOS/nixpkgs/pull/82408 still exist, sadly..

**Comment by veprbl** [comment]
> LGTM, but problems listed here #82408 still exist, sadly..

Yes, I saw that list. I could only reproduce the first one, but it is present in the 11.1.0.9080 as well. Feel free to open an issue for those, but, ideally, one would need to check with officially supported systems to see if it's an upstream issue.

**Comment by Ninlives** [comment]
@veprbl Skin Center works? Do you think that's related to some qt plugins installed in the global QT_PLUGIN_PATH?

**Comment by Ninlives** [comment]
No clue though, I didn't find any related logs..

**Pull request merged**